rules/library.bzl

```diff
@@ -952,6 +953,8 @@ def apple_library(
         generated_swift_header_name = module_name + "-Swift.h"
 
         if module_map:
+            # TODO: now that we always add module_map as a swiftc_input,
```

Comment:
We should move the TODO to line 967.

Reply:
yup removing in my next PR trying to get CI sandbox run